lib/python/bind_slice_methods.h

```diff
@@ -270,4 +288,9 @@ void bind_slice_methods(pybind11::class_<T, Ignored...> &c) {
       return out;
     });
   }
+  c.def("__getitem__", [](T &self, const std::vector<scipp::index> &indices) {
```

Comment:
Does this support numpy arrays as index lists? Would be useful

Reply:
Yes, added a test now.